Controls/YourProfileWindow.java

```diff
@@ -0,0 +1,95 @@
+package com.example.sportoAiksteliuRezervacija.fxControllers;
+
+import com.example.sportoAiksteliuRezervacija.StartGui;
+import com.example.sportoAiksteliuRezervacija.ds.User;
+import com.example.sportoAiksteliuRezervacija.hibernateControllers.UserHibControl;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.stage.Stage;
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class YourProfileWindow implements Initializable {
+
+    @FXML
+    public TextField nameField;
+    @FXML
+    public TextArea information;
+    @FXML
+    public TextField email;
+    @FXML
+    public ImageView imageView;
+    User user;
+    private int userId;
+
+    Image image;
+
+    EntityManagerFactory entityManagerFactory = Persistence.createEntityManagerFactory("CourseSystemMng");
+    UserHibControl userHibControl = new UserHibControl(entityManagerFactory);
+
+
+    @Override
+    public void initialize(URL url, ResourceBundle resourceBundle) {

```

Comment:
Metodo parametrai nenaudojami metode. Galbūt egzistuoja override'inamas metoas be šių argumentų? Jei taip, patarčiau tokį ir naudoti